packages/nimble/R/BUGS_modelDef.R

```diff
@@ -805,7 +808,11 @@ modelDefClass$methods(liftExpressionArgs = function() {
                 requireNewAndUniqueDecl <- any(contexts[[BUGSdecl$contextID]]$indexVarNames %in% all.vars(paramExpr))
                 uniquePiece <- if(requireNewAndUniqueDecl) paste0("_L", BUGSdecl$sourceLineNumber) else ""
                 newNodeNameExpr <- as.name(paste0('lifted_', Rname2CppName(paramExpr, colonsOK = TRUE), uniquePiece))   ## create the name of the new node ##nameMashup
-                newNodeNameExprIndexed <- addNecessaryIndexingToNewNode(newNodeNameExpr, paramExpr, contexts[[BUGSdecl$contextID]]$indexVarExprs)  ## add indexing if necessary
+                if(deparse(paramExpr[[1]]) %in% liftedCallsDoNotAddIndexing) {   ## skip adding indexing to mixed-size calls
```

Comment:
`deparse` is expensive.  I see that inside `isExprLiftable` there are multiple implied deparses (or `as.character`s, but in general `deparse` is safer).  Can we either assume `as.character` is valid (since it works in `isExprLiftable`) or *once* do `deparse(paramExpr[[1]]` and use the result in these various places?